utils/io.go

```diff
@@ -36,6 +41,12 @@ func ReadParamsAndOutputFromExt(path string) ([]string, error) {
 		}
 	}
 
+	// explicitly handle line that's too long
+	if errors.Is(scanner.Err(), bufio.ErrTooLong) {
```

Comment:
@dpastoor made this change you requested (and the small change in `notGradientMethods` above). Ready for you to take another (final?) look.

This passes it back out and it's eventually surfaced on line 62 or 106 of `cmd/summary.go`. I commented out lines 25 and 26 (the fix) above and ran the problem model and got:
```
$ bbi nonmem summary /Users/SethG/Downloads/70/70.lst 

FATA[0000] Attempting to parse /Users/SethG/Downloads/70/70.ext with ReadParamsAndOutputFromExt(): bufio.Scanner: token too long 

Process finished with exit code 1
```
which, you're right, is much better than panicing.

Reply:
cool looks good